nemo_skills/dataset/gpqa-es-prompt-es/__init__.py

```
@@ -0,0 +1,23 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Ruff (0.14.14)

The file is executable but no shebang is present

(EXE002)